content/en/installation/multioverview.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Multi-node Installation"
+keywords: 'Multi-node, Installation, KubeSphere'
+description: 'Multi-node Installation Overview'
+---
+
+In a production environment, a single-node cluster may not satisfy most of the needs as the cluster has limited resources with insufficient compute capabilities. Thus, single-node clusters are not recommended for large-scale data processing. Besides, a cluster of this kind is not available with high availability as it only has one node. On the other hand, a multi-node architecture is the most common and preferred choice in terms of application deployment and distribution.
+
+This section gives you an overview of multi-node installation, including the concept, scenarios, and KubeSphere installers (KubeKey and ks-installer). For information about the specific step of multi-node installation, see Installing on Public Cloud and Installing on On-premises Environment.
+
+## Concept
+
+A multi-node cluster is composed of at least one master node and one worker node. You can add additional nodes based on your needs (e.g. for high availability) both before and after the installation.
+
+- **Master**. A master node generally hosts the control plane that controls and manages the whole system.
+- **Worker**. Worker nodes run the actual applications deployed on them.
+
+## Scenarios
+
+A multi-node architecture is extremely useful in a variety of scenarios. A couple of examples are given as follows for your reference.
+
+### High Availability (HA)
+
+For single-node clusters, users will not be able to access data stored in the cluster once the node goes down. For highly available clusters with multiple nodes, the architecture features a failover system that will allow other nodes to take over in case of a node failure. This is extremely important to replicated apps as the scheduler can assign each replica to a different node if you have enough available ones. If a node fails, your app still functions. In other words, your services can run consistently.
```

Comment:
If you introduce HA, do not forget to take a look at this [section](https://kubesphere.io/docs/installation/intro/#high-availability-installation-for-production-environment)

Reply:
HA will not be introduced here. This part just briefly talks about one of the possible use cases.